instrumentation/jmx-metrics/library/src/main/resources/jmx/rules/tomcat.yaml

```
@@ -0,0 +1,82 @@
+---
+# For Tomcat, the default JMX domain is "Catalina:", however with some deployments like embedded in spring-boot
+# we can have the "Tomcat:" domain used, thus we use both MBean names for the metrics.
+
+rules:
+  - beans:
+    - Catalina:type=GlobalRequestProcessor,name=*
+    - Tomcat:type=GlobalRequestProcessor,name=*
+    prefix: tomcat.
+    metricAttribute:
+      tomcat.request_processor.name: param(name)
+    mapping:
+      errorCount:
+        metric: error.count
+        type: counter
+        unit: "{error}"
+        desc: The number of errors.
+      requestCount:
+        metric: request.count
+        type: counter
+        unit: "{request}"
+        desc: The number of requests processed.
+      maxTime:
+        metric: request.duration.max
+        type: gauge
+        sourceUnit: ms
+        unit: s
+        desc: The longest request processing time.
+      processingTime:
+        metric: request.processing_time
+        type: counter
+        sourceUnit: ms
+        unit: s
+        desc: Total time for processing all requests.
+      bytesReceived:
+        metric: &metric network.io
+        type: &type counter
+        unit: &unit By
+        desc: &desc The number of bytes transmitted.
+        metricAttribute:
+          tomcat.network.io.direction: const(received)
+      bytesSent:
+        metric: *metric
+        type: *type
+        unit: *unit
+        desc: *desc
+        metricAttribute:
+          tomcat.network.io.direction: const(sent)
```

Here we can use existing semconv [`network.io.direction`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/network/#network-io-direction) directly without introducing tomcat-specific attributes.

```suggestion
        metricAttribute:
          tomcat.network.io.direction: const(receive)
      bytesSent:
        metric: *metric
        type: *type
        unit: *unit
        desc: *desc
        metricAttribute:
          network.io.direction: const(transmit)
```

---

Good point, thanks!